Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -515,7 +515,15 @@ void setKValues(int type,float* values){
     for(i=0; i<7; i++){
        setGain(Kchannel[i],type,values[i]);
     }
-};
+}
+
+void setGains(int* type, float* values){
+    int gains[3] = {GAIN_KD,GAIN_KP,GAIN_KI};
```

Comment:
Set this as a global constant so you aren't reallocating every time it is called.